tests/node/node.go

```diff
@@ -21,44 +21,55 @@ package node
 
 import (
 	"fmt"
-	"os/exec"
 	"strings"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+
+	"github.com/k8snetworkplumbingwg/ovs-cni/tests/cmd"
 )
 
-// RunOnNode performs bash shell command on node
 // TODO: Use job with a node affinity instead
-func RunOnNode(node string, command string) (string, error) {
-	out, err := exec.Command("bash", "-c", fmt.Sprintf("docker ps | grep %s | awk '{ print $1}'", node)).CombinedOutput()
-	if err != nil {
-		panic(fmt.Sprintf("failed to run docker ps error output: %s", string(out)))
+func runAtNodeWithExtras(node string, quiet bool, command ...string) (string, error) {
```

Comment:
Once you drop unused RunQuietAtNode, the quiet flag is no longer needed

Reply:
ditto